python/rmm/docs/conf.py

```diff
@@ -89,6 +89,16 @@
 # This patterns also effect to html_static_path and html_extra_path
 exclude_patterns = []
 
+# List of warnings to suppress
+suppress_warnings = []
+
+# if the file deprecated.xml does not exist in the doxygen xml output, breathe
+# will fail to build the docs, so we conditionally add "breathe.rst" to the
```

Comment:
```suggestion
# will fail to build the docs, so we conditionally add "deprecated.rst" to the
```

Reply:
Haha copilot did that! (It actually made the same mistake in the code)